_source/logzio_collections/_prometheus-sources/telegraf-pgbouncer.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Ship PgBouncer metrics via Telegraf
+logo:
+  logofile: pgbouncer-telegraf.png
+  orientation: vertical
+data-source: PgBouncer metrics over Telegraf
+templates: ["docker"]
+contributors:
+  - daniel-tk
+  - nshishkin
+shipping-tags:  
+  - prometheus
+order: 800
+---
+
+
+## Overview
+
+Telegraf is a plug-in driven server agent for collecting and sending metrics and events from databases, systems and IoT sensors.
+
+To send your Prometheus-format PgBouncer metrics to Logz.io, you need to add the **inputs.pgbouncer** and **outputs.http** plug-ins to your Telegraf configuration file.
+
+#### Configuring Telegraf to send your metrics data to Logz.io
```

Comment:
need to tell customers to install pgbouncer as a prerequisite and only for linux